### DISCUSSION TITLE
How to connect 3.X agents to Azure DevOps 2022 latest update

### DISCUSSION THREAD

**Discussion by soroshsabz** [microsoft/azure-pipelines-agent#4412] [Q&A]
ITNOA

Hi

I need to have CI/CD on Ubuntu 22.04, and as I know, I could not install 2.X agents on Ubuntu 22.04 and in another side, I know Microsoft does not official support 3.X agents for Azure DevOps 2022.

So my question is, how to resolve my problem? is there any way to achieve CI/CD on Ubuntu 22.04 for Azure DevOps 2022?

thanks a lot

**Comment by svbrburgman** [comment]
I'm using windows agents 3.x without any issues with ADS(erver) 2022 for a couple of months now. 
I would just go for the 3.x (stable!) releases.

**Reply by soroshsabz** [reply]
So What did you think that 3.X agents does not connect to Azure DevOps 2022 server on Ubuntu 22.04?

Did you need some logs or etc for better troubleshooting?

thanks
